src/test/java/gololang/PredefinedTest.java

```diff
@@ -230,6 +230,13 @@ public void test_isArray() {
   @Test
   public void test_arrayOfType() throws ClassNotFoundException {
     assertThat((Class) Predefined.arrayTypeOf(Object.class), sameInstance((Class) Object[].class));
+    assertThat((Class) Predefined.arrayTypeOf(byte.class), sameInstance((Class) byte[].class));
+    try {
+      Predefined.arrayTypeOf(new Object());
+      Assert.fail();
+    } catch (Throwable t) {
+      assertThat((Class) t.getClass(), sameInstance((Class) AssertionError.class));
```

Comment:
I don't want create a test case juste for one assertion (seems to me not consistent with the test suite here), and use of `expect exception` for the last assertion can be confusing for the rest of the test case.